echo/server/dembrane/audio_lightrag/tests/e2e_test.py

```
@@ -0,0 +1,21 @@
+import os
+
+import pandas as pd
+import pytest
+
+from dembrane.config import AUDIO_LIGHTRAG_OUTPUT_JSON_FILEPATH
+from dembrane.audio_lightrag.main.run_etl import run_etl_pipeline
+
+
+@pytest.mark.usefixtures("conversation_df", "project_df")
+def test_run_etl_pipeline(conversation_df: pd.DataFrame, 
```

Ruff (0.8.2)

Unused function argument: `conversation_df`

(ARG001)